packages/iTwinUI-react/src/core/utils/components/VirtualScroll.tsx

```diff
@@ -100,20 +120,65 @@ export type VirtualScrollProps = {
 export const VirtualScroll = React.forwardRef<
   HTMLDivElement,
   VirtualScrollProps
->(({ itemsLength, itemRenderer, bufferSize = 10, style, ...rest }, ref) => {
+>((props, ref) => {
+  const { innerProps, outerProps, visibleChildren } = useVirtualization(props);
+
+  return (
+    <div {...outerProps} ref={ref}>
+      <div {...innerProps}>{visibleChildren}</div>
+    </div>
+  );
+});
+
+/**
+ * `useVirtualization` hook is using `VirtualScroll` component to render a huge amount of items.
```

Comment:
This wording looks wrong. `useVirtualization` is not using `VirtualScroll`, it's the other way around.

You could skip any mention of `VirtualScroll` and just explain what it does.
```suggestion
 * `useVirtualization` hook is used for efficiently rendering only the visible rows from a large list.
```

Reply:
Changed